### DISCUSSION TITLE
OSINT Tools

### DISCUSSION THREAD

**Discussion by jlengelbrecht** [CorentinTh/it-tools#725] [Ideas]
Would be cool to add a section for Cyber security folks. Maybe some things like built in phone infoga for gather details on a phone number. Or other tools to gather information needed for a pen test etc.

**Comment by gitmotion** [comment]
@jlengelbrecht seems like a lot to add to this but there's already an osint tool called [web-check](https://web-check.xyz/): https://github.com/Lissy93/web-check
- might be worth getting it added there
